src/Symfony/Component/Notifier/Bridge/FortySixElks/FortySixElksTransportFactory.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\FortySixElks;
+
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+
+/**
+ * @author Jon Gotlin <jon@jon.se>
+ */
+final class FortySixElksTransportFactory extends AbstractTransportFactory
+{
+    public function create(Dsn $dsn): FortySixElksTransport
+    {
+        if ('fortysixelks' !== $dsn->getScheme()) {
+            throw new UnsupportedSchemeException($dsn, 'fortysixelks', $this->getSupportedSchemes());
+        }
+
+        $host = 'default' === $dsn->getHost() ? null : $dsn->getHost();
+        $from = $dsn->getRequiredOption('from');
+
+        return (new FortySixElksTransport($this->getUser($dsn), $this->getPassword($dsn), $from, $this->client, $this->dispatcher))->setHost($host)->setPort($dsn->getPort());
+    }
+
+    protected function getSupportedSchemes(): array
+    {
+        return ['fortysixelks'];
```

Comment:
Wouldn't it be more readable to use `forty-six-elks` instead?

Reply:
No other transport schemas are named with dashes. Agree with your comment but named it similar to the others.

Reply:
Agreed! Renamed.